source/set-up/shared-channels.rst

```diff
@@ -0,0 +1,193 @@
+
+Extending the Invitation
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Copy the invitation code blob in the System message, then send the code blob and the decryption password to the remote Mattermost System Admin you want to securely connect with.
+
+2. Ensure that the remote Mattermost instance can access your workspace URL listed in the System message.
+
+Removing a Secure Connection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the following slash command to remove a secure connection from your Mattermost instance:
+
+``/secure-connection remove --connectionID``
+
+For example:
+
+``/secure-connection remove --connectionID AcmeUS``
+
+This slash command severs the trust relationship between the local Mattermost server and a remote Mattermost server, and removes the secure connection, AcmeUS, from all shared Mattermost channels.
+
+.. tip:: 
+
+    If AcmeUS was the only secure connection established, a System message would notify System Admins that the last secure connection has been removed, and the current channel is no longer shared.
+
+
+Reviewing Secure Connection Status
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the following slash command to review the current status of all secure connections established for your Mattermost instance:
+
+``/secure-connection status``
+
+Status details include:
+- Connection ID
+- Connection URL
+- Description
+- Invite accepted (Yes/No)
+- Online (Yes/No)
+- Last ping timestamp (UTC)
+
+Accepting a Secure Connection Invitation
+----------------------------------------
+
+Use the following slash command to accept a secure connection invitation from a remote Mattermost instance:
+
+``/secure-connection accept --name --displayname --password --invite [code blob]``
+
+For example:
+
+``/secure-connection accept --name AcmeUS --displayname “AcmeUSA” --password examplepassword --invite [code-blob]``
+
+This slash command accepts a secure connection invitation from AcmeUS.
+
+Sharing Channels with Secure Connections
+----------------------------------------
+
+Within a specific Public or Private channel, use the following slash command to invite secure connections:
+
+``/share-channel invite --connectionID <--readonly>``
+
+You can extend an invitation that permits remote members to participate in the channel based on their channel and member permissions. 
+
+Alternatively, you can extend a read-only invitation to a secure connection by appending the optional --readonly parameter to this command. Remote members can’t post or reply to messages within shared read-only channels.
+
+.. tip:: 
+
+    You can re-extend an invitation to a secure connection by omitting the optional ``--readonly`` parameter that also enables remote member participation in that channel.
+
+For example:
+
+``/share-channel invite --connectionID "AcmeUS"``
```

Comment:
@wiggin77 Is this also a case where ``--connectionID`` uses a GUID rather than name that should be corrected?

Reply:
Yes